### PR TITLE
fix: release workflow breaking

### PR DESCRIPTION
**Pull request by rudouglas**
Escaped quotation characters
Run the github release command inside the script to eliminate issues with strings between Python and Bash

### CODE REVIEW

File: scripts/actions/release-notes.py [hunk 1]
```diff
@@ -13,11 +13,14 @@
 result = "## :rocket: What's new?\n\n\n"
 
 # Get the Docs repo
-repo = github.get_repo("newrelic/docs-website")
+repo = github.get_repo("rudouglas/docs-website")
```
Comment (aswanson-nr) [inline scripts/actions/release-notes.py:16]: Reminder to update this before merging!

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Looks good to me! Just saw one field that needs to be updated

File: scripts/actions/release-notes.py [hunk 2]
```diff
@@ -32,7 +35,11 @@
   except AttributeError:
     pass
 
+# Limit the characters in the notes to 25000 as per api restrictions
+result = result[0:24999]
+
+# Encode the string to escape characters
+result = result.replace('"','\\"')
```
Comment (nr-kkenney) [inline scripts/actions/release-notes.py:42]: i think you may wanna swap these lines. if we replace one character with two, that could send us over the limit again.

File: scripts/actions/release-notes.py (hunk 1 above)
Comment (rudouglas) [reply scripts/actions/release-notes.py]: 🤦

File: scripts/actions/release-notes.py (hunk 2 above)
Comment (rudouglas) [reply scripts/actions/release-notes.py]: Ah yes i was trying to be clever and not have to replace characters if they were over a certain limit already 🥇

**Review by nr-kkenney — APPROVED** [review]

**Pull request merged**